languagetool-core/src/main/java/org/languagetool/UserConfig.java

```diff
@@ -104,12 +105,13 @@ public UserConfig(List<String> userSpecificSpellerWords,
                     LinguServices linguServices, boolean filterDictionaryMatches,
                     @Nullable List<String> abTest, @Nullable Long textSessionId,
                     boolean hidePremiumMatches, List<String> preferredLanguages) {
-    this(userSpecificSpellerWords, userSpecificRules, ruleValues, maxSpellingSuggestions, premiumUid, userDictName, userDictCacheSize, linguServices, filterDictionaryMatches, abTest, textSessionId, hidePremiumMatches, preferredLanguages, false, false, false);
+    this(userSpecificSpellerWords, userSpecificRules, ruleValues, true, maxSpellingSuggestions, premiumUid, userDictName, userDictCacheSize, linguServices, filterDictionaryMatches, abTest, textSessionId, hidePremiumMatches, preferredLanguages, false, false, false);
   }
 
   public UserConfig(List<String> userSpecificSpellerWords,
                     List<Rule> userSpecificRules,
                     Map<String, Object[]> ruleValues,
+                    boolean suggestionsEnabled,
```

Comment:
Instead of adding the param to the existing constructor, please overload. I think we are using this one in other projects, too.